mantidproject/mantid
# Mantid Qt Icons Library for Workbench

**Description of work.**
The aim of this PR is to replace our dependency on QtAwesome (Python library) and use our own similar library which is available in C++ and Python. The basic implementation, is that we use the font, and the respective unicode to draw a icon wherever it is needed. This approach allows us to use many different fonts at the same time. It should be quite easy to add support for another icon/font library in Mantid. At present the only one that is intended for support is the Material Design Icons.

**To test:**
- Open Workbench
	- Check the play, stop, and new tab button's have their icon.
- Open a Plot
	- Check all buttons except Fit have an Icon including the new Grid Icon
- Open a MatrixWorkspaceTableView aka Show Data on a MatrixWorkspace
	- Right click on a row and see the chart and copy icons.
- Open a TableWorkspaceView aka Show Data on a Table Workspace
	- Rick click on a Row/Column and see the icons are loaded properly.
- Open Interfaces->SANS->ISIS SANS
	- Check all the buttons above the table except Export Table have an Icon, all of the tabs on the left of the GUI should also have an Icon on them. 



*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
